cfme/tests/infrastructure/test_snapshot.py

```diff
@@ -97,6 +97,7 @@ def test_memory_checkbox(small_test_vm, provider, soft_assert):
 
 
 @pytest.mark.rhv1
+@pytest.mark.meta(automates=[1608475])
 def test_snapshot_crud(small_test_vm, provider):
```

Comment:
Please add BZ(1608475) in doc string under ```Bugzilla``` tag

Reply:
@ganeshhubale this is not required

Reply:
@ganeshhubale I discussed this offline with @lina-nikiforova and @john-dupuy, John  will be putting together guidelines PR to discuss it further.